src/devices/bus/spectrum/neogs.cpp

```diff
@@ -90,27 +104,44 @@ class neogs_device : public device_t, public device_zxbus_card_interface
 	memory_bank_creator m_bank_rom;
 	memory_bank_creator m_bank_ram;
 	memory_view m_view;
-	required_device_array<dac_word_interface, 8> m_dac;
+	required_device_array<dac_word_interface, 2> m_dac;
+	required_device<spi_sdcard_sdhc_device> m_sdcard;
 
 private:
+	TIMER_CALLBACK_MEMBER(spi_clock);
+	emu_timer *m_spi_clock;
+	int m_spi_clock_cycles;
+	bool m_spi_clock_state;
+
 	template <u8 Bank> u8 ram_bank_r(offs_t offset);
```

Comment:
Please put these data members with the other data members in the `private` section below, rather than mixing them with member functions like this.